### DISCUSSION TITLE
migrating from discordpy

### DISCUSSION THREAD

**Discussion by Mineinjava** [nextcord/nextcord#290] [Q&A] [answered]
having a migration guide from discordpy to nextcord would be appreciated

**Comment by TAG-Epic** [comment, marked as answer]
See https://nextcord.readthedocs.io/en/latest/migrating_2.html#migrating-2-0

**Reply by veni-vidi-code** [reply]
see here aswell https://nextcord.readthedocs.io/en/latest/migrating_to_nextcord.html
